test/evergreen.yml

```diff
@@ -1954,6 +1954,21 @@ tasks:
           format_test_script_args: -S
           times: 16
 
+  - name: race-condition-stress-sanitizer-test
+    # set an evergreen timeout of 25 hours
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC="/opt/mongodbtoolchain/v3/bin/clang -fsanitize=address" PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC -fno-omit-frame-pointer"
```

Comment:
I think "-fsanitize=address" option should be part of CFLAGS. Just curious to know the reason for not setting CXX variable.

Reply:
`CXX` was not set in Jenkins and isn't relevant to the test so I opted to keep the task definition as clean as possible.